classifiers/text_analysis/emotionality_detection/config.py

```diff
@@ -19,28 +19,31 @@ def get_config():
             "text_analysis",
```

Comment:
Would add this to the "llm" category as well as this is using a roberta model from HuggingFace.